docs/source/core_services/service_agents/actuator/ActuatorScheduleResponse.rst

```diff
@@ -10,7 +10,7 @@ ActuatorAgent will respond on the topic
     #python
     {
         'type': <'NEW_SCHEDULE', 'CANCEL_SCHEDULE'>
-        'requesterID': <Agent ID from the request>,
+        'requesterID': <Agent VIP identity from the request>,
```

Comment:
Is the requesterID required?  Perhaps mark as legacy or something and not include. @jhaack thoughts on this?